### PR TITLE
now generates player data per realm, guild data uses this data

### PR DESCRIPTION
**Pull request by OliverRadini**
Should pave the way for guild pages being able to list player data.

### CODE REVIEW

File: herald/generate-player-data.js [hunk 1]
```diff
@@ -133,14 +132,40 @@ let hibernia =
...
+        raceName: "briton",
+        className: "cleric",
+        level: 10,
+        xpPercent: 1,
+        rpPercent: 1,
+        realmRank: 1,
+        realm: 'albion',
+});
+
+let playerNames.push('Testplayer');
```
Comment (burfo) [inline herald/generate-player-data.js]: This is actually an error.  You can't redeclare `playerNames`, it's already declared on line 144.

**Review by burfo — CHANGES_REQUESTED** [review]
I'm fine with merging this, but there's actually 1 error that you probably should fix.   :)

File: herald/generate-player-data.js (hunk 1 above)
Comment (OliverRadini) [reply herald/generate-player-data.js]: I'm suprised that worked at all, it's just a typo really, there's no need for `let` there at all. It's not actually redeclaring playerNames I don't think, but presumably declaring whatever `playerNames.push` evaluates to.

In any case I can fix this later on today.

Comment (burfo) [reply herald/generate-player-data.js]: Well that's the thing, it didn't work for me.  It errored with a SyntaxError at runtime.
```
\herald\generate-player-data.js:163
let playerNames.push('Testplayer');
    ^^^^^^^^^^^
SyntaxError: Identifier 'playerNames' has already been declared
    at Object.exports.runInThisContext (vm.js:78:16)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:420:7)
    at startup (bootstrap_node.js:139:9)
    at bootstrap_node.js:535:3
```
Thanks for fixing it!

Comment (OliverRadini) [reply herald/generate-player-data.js]: Ah interesting, perhaps I may have edited it by mistake after commiting, hopefully that'll be ifne now

**Pull request merged**